src/main/java/io/logz/jmx2logzio/listener/prometheus/PromLogzioSender.kt

```diff
@@ -0,0 +1,128 @@
+package io.logz.jmx2logzio.listener.prometheus
+
+import io.logz.prometheus.protobuf.Types
+import io.logz.sender.HttpsRequestConfiguration
+import io.logz.sender.SenderStatusReporter
+import io.logz.sender.exceptions.LogzioServerErrorException
+import java.util.concurrent.ExecutionException
+import java.util.concurrent.Executors
+import java.util.concurrent.ScheduledExecutorService
+import java.util.concurrent.TimeUnit
+import java.util.concurrent.TimeoutException
+import java.util.concurrent.atomic.AtomicBoolean
+
+class PromLogzioSender(
+    private val reporter: SenderStatusReporter,
+    private val tasksExecutor: ScheduledExecutorService,
+    private val timeSeriesQueue: TimeSeriesDiskQueue,
+    private val debug: Boolean = false,
+    private val drainTimeoutSec: Long = 5,
+    httpsRequestConfiguration: HttpsRequestConfiguration
+) {
+    companion object {
+        private const val MAX_SIZE_IN_BYTES = 3 * 1024 * 1024 // 3 MB
```

Comment:
Max size of what?

Reply:
it's a secret the author of logzio-sender didn't wan't us to know :D